includes/admin/functions.php

```diff
@@ -106,6 +106,11 @@ function amp_add_options_menu() {
 }
 add_action( 'wp_loaded', 'amp_add_options_menu' );
 
+/**
```

Comment:
Missing function description, param description, and return description. We've also been adding `@since` tags.

Reply:
I do not understand that function's parameter well enough to fully document so I backed out the partial PHPDoc I had added.